src/nextlabs_sdk/_cloudaz/_engine/_request_plan.py

```
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class RequestPlan:
+    """A declarative description of a single HTTP request to issue."""
+
+    method: str
+    path: str
+    params: dict[str, int] | None = None  # noqa: WPS110
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: WPS110

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)